engine/server/engine/centralized_logs/client_implementations/persistent_volume/persistent_volume_logs_database_client_test.go

```diff
@@ -256,12 +255,12 @@ func TestStreamUserServiceLogs_ThousandsOfLogLinesSuccessfulExecution(t *testing
 		testUserService1Uuid: true,
 	}
 
-	file1 := fmt.Sprintf(volume_consts.PerFileFmtStr, volume_consts.LogsStorageDirpathForTests, string(enclaveUuid), testUserService1Uuid, volume_consts.Filetype)
-	underlyingFs := &fstest.MapFS{
-		file1: {
-			Data: []byte(logLinesStr),
-		},
-	}
+	underlyingFs := volume_filesystem.NewMockedVolumeFilesystem()
+
+	file1PathStr := fmt.Sprintf(volume_consts.PerFileFmtStr, volume_consts.LogsStorageDirpath, string(enclaveUuid), testUserService1Uuid, volume_consts.Filetype)
```

Comment:
@tedim52 I don't think this is wrong per se, but a way to add another level of checking would be to write a literal string version of this code so that you know that the "coded" version of this (in `getFilepathStr`) is producing the string that you expect)